src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +101,20 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate localDate = LocalDate.now();
+        return localDate.isBefore(someDate) ? (someDate + " is after " + localDate)
+                : (localDate.isAfter(someDate) ? (someDate + " is before " + localDate)
+                : (someDate + " is today"));
     }
 
     /**
      * Дана дата в строковом формате и временная зона.
      * Верните LocalDateTime в этой временной зоне.
+     *
      * @return LocalDateTime
      */
     public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone) {
-        return LocalDateTime.now();
+        return ZonedDateTime.parse(dateInString + "[" + zone + "]").toLocalDateTime();
```

Comment:
let's try to be less straightforward and solve this with the help of Instant class